doc/cli/operator-sdk_alpha_run.md

```diff
@@ -0,0 +1,31 @@
+## operator-sdk alpha run
+
+Run an Operator in a variety of environments
+
+### Synopsis
+
+Run an Operator in a variety of environments
+
+```
+operator-sdk alpha run [flags]
+```
+
+### Options
+
+```
+      --force-registry            Force deletion of the in-cluster registry. This option is a no-op on 'up'.
```

Comment:
Curious about this: `This option is a no-op on 'up'.`

Does `up` need to be `run`? But that would make it seem like it's always a no-op. I'm a little confused...

Reply:
This flag should be specific to `cleanup` so I've moved it to `cleanup.NewCmd()`.